_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,87 @@
+{% if page.name == "tinymcespellchecker.md" %}
+### `content_langs`
+{% else %}
+## `content_langs`
+{% endif %}
+
+{{ site.requires_5_9v }}
+
+This option sets the languages displayed in the `language` dropdown toolbar button and the `language` menu item. The option accepts an array of language objects. Each language object is a JavaScript object defined using the following properties.
```

Comment:
```suggestion
This option sets the languages displayed in the `language` toolbar button and the `language` menu item. The option accepts an array of language specs. Each language spec is a JavaScript object defined using the following properties.
```
Dropdown is necessary here. We're also repeating `objects` too much here and it could be confusing since it's worded in such a way they are different, so tried to come up with another name.